distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+As most configurations are stored in configuration files, in some instances it may make sense to edit those configuration files directly.
+
+==== Configuring ${branding} Global Settings
+
+Global configuration settings are configured via the properties file `system.properties`.
+These properties can be manually set by editing this file or set via the initial configuration from the ${admin-console}.
+
+.Global Settings
+[cols="2,3m,2,3,3m,1"]
+|===
+|Title
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+6+^h|Keystore and truststore java properties
+
+|Keystore
+|javax.net.ssl.keyStore
+|String
+|Path to server keystore
+|etc/keystores/serverKeystore.jks
+|Yes
+
+|Keystore Password
+|javax.net.ssl.keyStorePassword
+|String
+|Password for accessing keystore
+|changeit
+|Yes
+
+|Truststore
+|Truststorejavax.net.ssl.trustStore
+|String
+|The trust store used for SSL/TLS connections. Path is relative to `<INSTALL_HOME>`.
+|etc/keystores/serverTruststore.jks
+|Yes
+
+|Truststore Password
+|javax.net.ssl.trustStorePassword
+|String
+|Password for server Truststore
+|changeit
+|Yes
+
+|Keystore Type
+|javax.net.ssl.keyStoreType
+|String
+|File extension to use with server keystore
+|jks
+|Yes
+
+|Truststore Type
+|javax.net.ssl.trustStoreType
+|String
+|File extension to use with server truststore
+|jks
+|Yes
+
```

Comment:
The Admin Console has a `Private Key Password` field which isn't present in the system.properties file but should be documented somewhere.